doc/reference/cardano-governance.md

```diff
@@ -0,0 +1,235 @@
+# Cardano governance
+
+Until the [Voltaire era](https://roadmap.cardano.org/en/voltaire/) is released, Cardano has a federated governance mechanism that allows updating the protocol parameters. Such updates are done using an _update proposal_ mechanism. In the current setup, only the holders of the _genesis keys_ can submit and vote on proposals.
+
+Update proposals can be used to introduce changes to the values of specific parameters (e.g., the maximum block size, the maximum transaction size etc.), changes to the validation rules at any level (transaction, block, or blockchain), or even changes at the consensus protocol itself.
+
+Protocol parameters are used to affect the operation of the Cardano protocol. They may be either **updatable** or **non-updatable**. Updatable parameters can be tuned to vary the operation of the block-producing protocol, impacting the proportion of pools that are federated/non-federated, influence of the 'pledge', etc. Non-updatable parameters affect the fundamentals of the blockchain protocol, which includes defining the genesis block, basic security properties etc. [CIP-9](https://cips.cardano.org/cips/cip9/) and [CIP-28](https://cips.cardano.org/cips/cip28/) detail which parameters are updatable and which are not.
```

Comment:
This terminology is at odds with the terminology that we use in the ledger (the code and the specs). In the ledger, **protocol parameters** are _all_ updatable. Values in the ledger that are hard coded we refer to as **global constants**. It might be fine to have different definitions, I'm just pointing out a potential confusion... I see now that the terminology used here is consistent with those two CIPs, so potential confusion already exists :)

Reply:
Perhaps we to amend the CIP-9 so that non-updatable parameters are referred to as global constants?

Reply:
That's a great idea!